packages/deployer/src/build/bundler.ts

```diff
@@ -15,6 +15,7 @@ export async function getInputOptions(
   analyzedBundleInfo: Awaited<ReturnType<typeof analyzeBundle>>,
   platform: 'node' | 'browser',
   env: Record<string, string> = { 'process.env.NODE_ENV': JSON.stringify('production') },
+  sourcemapEnabled: boolean = false,
```

Comment:
Let's make this an object so we can pass more bundler options in the future


```suggestion
  {sourcemap = false }: { sourcemap: boolean}
```

Reply:
Agreed! Updated in latest commit.